src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -441,6 +443,42 @@ public function get($id, $invalidBehavior = ContainerInterface::EXCEPTION_ON_INV
     }
 
     /**
+     * Get all services implementing a class or an interface.
+     *
+     * @param string $class
+     * @param bool   $reload whether or not to reload the type map
+     *
+     * @return string[]
+     */
+    public function getServiceIdsOfType($class, $reload = true)
```

Comment:
The controller definitions can be changed at any time so the type map has to be reloaded each time we use it. 
This parameter is here to prevent this reload in case we are sure it is useless.